hw/ip/rstmgr/rstmgr_reg.core

```diff
@@ -3,16 +3,15 @@ CAPI=2:
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 name: "lowrisc:ip:rstmgr_reg:0.1"
-description: "rstmgr registers"
+description: "Reset manager registers"
 
 filesets:
   files_rtl:
     depend:
       - lowrisc:tlul:headers
-      - fileset_top ? (lowrisc:systems:rstmgr_reg)
     files:
-      - fileset_ip ? (rtl/rstmgr_reg_pkg.sv)
-      - fileset_ip ? (rtl/rstmgr_reg_top.sv)
+      - rtl/rstmgr_reg_pkg.sv
```

Comment:
ah same rationale here regarding rstmgr.

Reply:
Yes I see, I thought it would simplify the dependencies if doing it in one core file only. But let's continue in the other discussion.